quipucords/api/migrations/0003_system_platform_id.py

```diff
@@ -32,7 +38,17 @@ def add_system_platform_id(apps, schema_editor):
                     # json dumps/loads changes type of dictionary
                     # removes massive memory growth for cached_fingerprints
                     cached_fingerprints.append(json.loads(json.dumps(serializer.data)))
+                    # Check if fingerprint has canonical facts
+                    for fact in CANONICAL_FACTS:
+                        if json.loads(json.dumps(serializer.data)).get(fact):
```

Comment:
Let's assign `json.loads(json.dumps(serializer.data))` to a variable.  This could be expensive.